sql/parse/parse_test.go

```diff
@@ -654,14 +654,70 @@ var fixtures = map[string]sql.Node{
 	),
 	`SET @@session.autocommit=1, foo="bar"`: plan.NewSet(
 		plan.SetVariable{
-			Name:  "autocommit",
+			Name:  "@@session.autocommit",
```

Comment:
this means that we are adding session variables as global variables (but with a name `@@session.blah`) now?

Reply:
Why the change adding `@@session.`? It should work without it too

Reply:
Because right now for `SetExpr` we don't have `Qualifiers` so in `convertSet` we return what was passed.

Reply:
But we should trim that part as it was before. Only session variables should be saved with `Set`